test/project.test.ts

```diff
@@ -142,12 +147,29 @@ describe('Default lambda runtime for CDK app and lib', () => {
   });
 
   test('CDK lib has default lambda runtime', () => {
-    const project = new GemeenteNijmegenCdkLib({ cdkVersion: '2.51.0', defaultReleaseBranch: 'main', name: 'test project', author: 'test', authorAddress: '', repositoryUrl: '' });
+    const project = new GemeenteNijmegenCdkLib({ cdkVersion: '2.51.0', defaultReleaseBranch: 'main', name: 'test project', author: 'test', authorAddress: '', repositoryUrl: '', repository });
```

Comment:
Wordt/werd de repository-url-waarde hier gebruikt? Voelt nu alsof 2x dezelfde param wordt gebruikt

Reply:
Die waarde is verplicht… Ik snap ook niet precies wat die doet. Maar als je ’m niet mee geeft gaat TS klagen.